index.html

```diff
@@ -0,0 +1,309 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <script src="https://code.jquery.com/jquery-3.6.0.min.js"></script>
+    <link rel="stylesheet" href="style.css" />
+    <script src="https://kit.fontawesome.com/3b9cd09213.js" crossorigin="anonymous"></script>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+</head>
+<body>
+    <header>
+        <div class="container">
+      <nav>
+        <div class="logo">
+          <h4>My logo</h4>
+        </div>
+        <ul class="nav_links">
+          <li><a href="#">Portfolio</a></li>
+          <li><a href="#">about</a></li>
+          <li><a href="#">contact</a></li>
+          <i class="fa-solid fa-bars"></i>
+        </ul>
+      </nav>
+      <div class="present">
+        <h1>I’m Drissa Glad to see you!</h1>
+        <p>
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hesitate to contact
+          me.
+        </p>
+        <h3>Let's connect</h3>
+        <div class="social">
+          <a href="#"><i class="fa-brands fa-twitter"></i></a>
+          <a href="#"><i class="fa-brands fa-linkedin"></i></a>
+          <a href="#"><i class="fa-brands fa-mastodon"></i></a>
+          <a href="#"><i class="fa-brands fa-github"></i></a>
+          <a href="#"><i class="fa-brands fa-angellist"></i></a>
+        </div>
```

Comment:
fixed

Reply:
[OPTIONAL] -  because it was not mentioned by the first reviewer.

The header should only contain the logo and the navigation, and not the headline.

Kindly refactor accordingly, so that the `h1`, the description, the social media links to be outside the `header`.